### PR TITLE
Workaround for #25

### LINKED ISSUE #25: error.sourceName must not be null
I'm trying to use the plugin, but I'm getting an error. 

My code in _build.gradle_ is:
```
plugins {
    id 'com.android.application'
    id "org.gradlewebtools.minify" version "1.2.1"
}
...
task jsMinify(type: org.gradlewebtools.minify.JsMinifyTask) {
    srcDir = project.file("$projectDir/src/main/assets/Token/app")
    dstDir = project.file("$projectDir/src/main/assets/Token/appmin")
    options.originalFileNames = true
    options.compilationLevel = com.google.javascript.jscomp.CompilationLevel.SIMPLE_OPTIMIZATIONS
}
```

The error I'm getting is:

```
Execution failed for task ':app:jsMinify'.
> error.sourceName must not be null

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:jsMinify'.
Caused by: java.lang.NullPointerException: error.sourceName must not be null
	at org.gradlewebtools.minify.minifier.result.Event.<init>(Event.kt:24)
	at org.gradlewebtools.minify.minifier.result.Error.<init>(Error.kt:8)
	at org.gradlewebtools.minify.minifier.js.JsMinifier.minifyFile(JsMinifier.kt:40)
	at org.gradlewebtools.minify.minifier.Minifier.minifyFileInternal(Minifier.kt:76)
	at org.gradlewebtools.minify.minifier.Minifier.handleFile(Minifier.kt:69)
	at org.gradlewebtools.minify.minifier.Minifier.minifyInternal(Minifier.kt:40)
	at org.gradlewebtools.minify.minifier.Minifier.minify(Minifier.kt:28)
	at org.gradlewebtools.minify.JsMinifyTask.minify(JsMinifyTask.kt:25)
```

Anyone could help me to bypass the problem?

### PR DESCRIPTION
**Pull request by timon-schelling**
#25 for more information
closes #25

### CODE REVIEW

File: src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt [hunk 1]
```diff
@@ -21,10 +21,10 @@ open class Event {
         charNo = error.charno
```
Comment (616slayer616) [inline src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]: Could this and the lineNumber create the same issue?

Comment (timon-schelling) [reply src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]: I guess everything from the "Java World", Yes in theory.

Comment (timon-schelling) [reply src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]: I try to look into Closure for that...

Comment (616slayer616) [reply src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]: I think if its only in theory we dont have to address it.
THe example code would help.

Comment (timon-schelling) [reply src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]:
```java
public abstract class JSError implements Serializable {

  /** A type of the error. */
  public abstract DiagnosticType getType();

  /** Description of the error. */
  public abstract String getDescription();

  /** Name of the source */
  @Nullable
  public abstract String getSourceName();

  /** One-indexed line number of the error location. */
  public abstract int getLineno();

  /** Zero-indexed character number of the error location. */
  public abstract int getCharno();
```
As you can see only "getSourceName" is marked  with @Nullable, I think we should therefor trust them not to return null for the other members like "getDescription" and "getLineno".

Comment (616slayer616) [reply src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]: Very nice documentation of them

**Review by 616slayer616 — APPROVED** [review]

File: src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt (hunk 1 above)
Comment (timon-schelling) [reply src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt:21]: > I think if its only in theory we dont have to address it.
> THe example code would help.

Yes, I think we can trust them following nullability annotation convention

**Pull request merged**